### ISSUE TITLE
loss function in eq5 goes negative infinity?

### ISSUE THREAD

**Issue by rowedenny** [aamir-mustafa/pcl-adversarial-defense#3]
I am interested in this paper, and I am curious that based on the PL definition in eq5, is it possible that it may go to negative infinity then?

**Comment by aamir-mustafa** [comment]
Thank you for your interest. In an extension of the work (under submission) we have explained in detail (qualitatively and quantitatively) what happens when the model is trained for a larger number of epochs and how the PCL loss decreases to a much larger extent.
